docs/modules/getting-started/pages/wan.adoc

```diff
@@ -0,0 +1,282 @@
+. Click on **Execute Query**; you will see the data you've put in. 
++
+image:wan/map-entries.png[Management Center's SQL Browser shows the map data]
+. Close the SQL browser.
+
+== Step 5. Verify the Replication
+
+With WAN Replication enabled, your `cities` map and its data should have been replicated from the active cluster (Tokyo)
+to the passive one (London). In this step, you'll verify that the `cities` map now also exists in the London cluster.
+
+. In Management Center, select `london` in the dropdown field left to "Cluster Connections" on top of the user interface.
+. Go to **Storage > Maps** and see that the `cities` map is there.
+
+Additionally, you can query the map entries on the London cluster using the SQL browser in Management Center.
+
+. You first need to create the mapping so that the London cluster can read the map entries.
+Open the SQL browser, type in the same `CREATE MAPPING` command from Step 4 in the editor, and **Execute Query**.
++
+image:wan/create-mapping-london.png[Create mapping on the London cluster]
+. In the `select a map` field, choose `cities`. The editor shows the default `SELECT * FROM cities;` query. Once you execute it, you will see the entries of `cities` map, as in Step 4.
+. Close the SQL browser.
+
+== Step 6. Update a Map Entry on the Active Cluster
+
+In this step, you'll update an entry in the `cities` map on the active cluster (Tokyo) and verify the update is replicated to the passive one (London).
+
+. Select `tokyo` in the dropdown field left to "Cluster Connections" on top of the user interface and go to **SQL Browser**.
+. Choose `cities` in the `select a map` field and execute the default `SELECT * FROM "cities"` query.
++
+image:wan/entry-tobe-modified.png[]
+. Now, you are going to modify the data shown above in the red box (the entry having the key `1`). In the SQL editor, delete the default query and type in the following command.
++
+[source,shell]
+----
+SINK INTO cities VALUES
+(1, 'Austria', 'Vienna');
+----
+Execute the query and see the entry has changed.
++
+image:entry-modified.png[Data is modified]
+. Close the SQL browser and now go to the passive cluster (London) by choosing `london` in the dropdown field left to "Cluster Connections" on top of the user interface.
+. Open the SQL browser, choose `cities` in the `select a map` field and execute the default query.
+. You will see the entry having the key `1` is also modified.
+
+In this step, you have seen that a data modification in the active cluster is immediately replicated to the passive one.
+If you do the modification first on the passive cluster, you'd see that the modification is not applied to the active one.
+
+== Step 7. Shut Down the Cluster
+
+Shut down the cluster you've created in this tutorial so that you can start a fresh one when you
+move to the other tutorials. To shutdown, close the terminals in which the members are running or press kbd:[Ctrl+C] in each terminal.
```

Comment:
```suggestion
move to the other tutorials. To shut down the cluster, close the terminals in which the members are running or press kbd:[Ctrl+C] in each terminal.
```

Reply:
Does closing the terminals work? I'm not sure if they continue running in the background.